download/community.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: default
+tab: download
+title: Flyway Community Download
+cssid: download-community
+---
+
+# Download Flyway Community
+
+Flyway is available in many different forms. Choose from the options below for specific instructions on how to download for the platform of your choice.
```

Comment:
Better wording would be 'available for many different platforms' instead of 'forms'.